gas/testsuite/gas/arc/taux.d

```diff
@@ -16,6 +16,7 @@
 .* U d2l
 .* U dpfp_status
 .* U fp_status
+.* U jli_base
```

Comment:
The test was failing after the jli_base AUX register was corrected for ARCv2. Does this not mark it as undefined for the arc700 so the test will pass without the AUX register showing up?